server/internal/infrastructure/gcp/cloudtasks/cloudtasks.go

```diff
@@ -0,0 +1,45 @@
+package cloudtasks
```

Comment:
Do not create more folders than you need. Keep the folder hierarchy as shallow as possible.

`gcp/cloudtask/cloudtask.go` -> `gcp/cloudtask.go`

Since this is just queuing the passed tasks, the amount of implementation should not be that large.